packages/algob/src/lib/compile.ts

```diff
@@ -23,9 +25,14 @@ export class CompileOp {
   // Will throw an exception if the source file doesn't exists.
   // @param filename: name of the TEAL code in `/assets` directory.
   //   (Examples: `mysc.teal, security/rbac.teal`)
-  //   MUST have a .teal extension
+  //   MUST have a .teal or .py extension
   // @param force: if true it will force recompilation even if the cache is up to date.
   async ensureCompiled (filename: string, force: boolean): Promise<ASCCache> {
+    if (filename.endsWith(pyExt)) {
+      const pyCompile = new PyCompileOp(this);
```

Comment:
Please move this to constructor and store the `pyCompiler` as a class field.

Reply:
ok